ui/admin/app/templates/scopes/scope/policies/index.hbs

```diff
@@ -105,7 +105,7 @@
                     {{/if}}
                     {{#if perms.canDelete}}
                       <dd.Interactive
-                        @text={{t 'resources.policy.actions.delete'}}
+                        @text={{t 'resources.policy.actions.detach'}}
```

Comment:
What about actually deleting a storage policy from the policy page? Isn't that actually deleting? 

<img width="2656" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/f422914a-9baf-45fd-98d5-7838466fb813">

Shouldn't this only be for detatching from scopes?

Reply:
ty, good catch, fixed now